src/pages/using-spark/components/card.mdx

```diff
@@ -0,0 +1,109 @@
+---
+  title: Card
+---
+import ComponentPreview from '../../../components/ComponentPreview';
+import { SprkDivider } from '@sparkdesignsystem/spark-react';
+
+# Card
+
+Card is a flexible container
+for grouping small chunks of related content.
+
+<ComponentPreview
+  componentName="card--default-story"
+  hasReact
+  hasAngular
+  hasHTML
+/>
+
+### Usage
+
+Card provides a way to present
+information quickly. While
+flexible in nature, they will
+typically have a heading and
+a call-to-action and be used in
+groups within a grid.
+
+### Guidelines
+
+- The content arrangement inside of a Card is intentionally flexible.
+- Cards have a maximum width of `425px`.
```

Comment:
I know this is content related, but could we update this to be 'Card has a maximum width...' to maintain consistency?